Reply/app/src/main/java/com/example/reply/ui/navigation/ReplyNavigationActions.kt

```diff
@@ -25,16 +25,15 @@ import androidx.compose.ui.graphics.vector.ImageVector
 import androidx.navigation.NavGraph.Companion.findStartDestination
 import androidx.navigation.NavHostController
 import com.example.reply.R
+import kotlinx.serialization.Serializable
 
-object ReplyRoute {
-    const val INBOX = "Inbox"
-    const val ARTICLES = "Articles"
-    const val DM = "DirectMessages"
-    const val GROUPS = "Groups"
-}
+@Serializable data object InboxRoute
```

Comment:
What about using a sealed class here? Might be useful for limiting what can be defined as a route?

Reply:
Good suggestion. Done.